src/components/include/cos_kernel_api.h

```diff
@@ -42,7 +42,7 @@ struct cos_compinfo {
 	struct cos_meminfo mi;	     /* only populated for the component with real memory */
 };
 
-void cos_compinfo_init(struct cos_compinfo *ci, captblcap_t pgtbl_cap, pgtblcap_t captbl_cap, compcap_t comp_cap, vaddr_t heap_ptr, capid_t cap_frontier, struct cos_compinfo *ci_resources);
+void cos_compinfo_init(struct cos_compinfo *ci, pgtblcap_t pgtbl_cap, captblcap_t captbl_cap, pgtblcap_t local_pgtbl_cap, compcap_t comp_cap, vaddr_t heap_ptr, capid_t cap_frontier, struct cos_compinfo *ci_resources);
```

Comment:
wow, this mistyping is horrible.  Thanks for catching it.  See above for local vs. global.